spacy/language.py

```diff
@@ -313,10 +287,7 @@ def create_pipe(self, name, config=dict()):
         DOCS: https://spacy.io/api/language#create_pipe
         """
         if name not in self.factories:
-            if name == "sbd":
-                raise KeyError(Errors.E108.format(name=name))
```

Comment:
Why remove these errors?

Reply:
Because they're only for backwards-compatibility handling of single very specific cases that haven't worked for a long time now. No user should have `sbd` in their pipeline anymore at this point, because it would have errored anyway. I don't think this specific case warrants extra treatment anymore – we might as well show the same error we show for all other non-existent factories.